### PR TITLE
Chunked checksums doc

### PR DESCRIPTION
**Pull request by drernie**
Separate PR for finalizing documentation

- changed filename
- removed pseudo-code
- added final registration data

### CODE REVIEW

File: docs/CHUNKED_CHECKSUMS.md [hunk 1]
```diff
@@ -1,9 +1,4 @@
-# sha2-256-chunked
-
-## DRAFT multihash codec 0xb510
-
-> Hash of concatenated SHA2-256 digests of 8*2^n MiB source chunks
-> where n = ceil(log2(source_size / (10^4 * 8MiB))
+# CHUNKED CHECKSUMS
 
 This variant of sha2-256 is designed to enable large files
 to be efficiently uploaded and hashed in parallel using fixed size chunks
```
Comment (sir-sigurd) [inline docs/CHUNKED_CHECKSUMS.md:4]: how about
```suggestion
to be efficiently uploaded and hashed in parallel using uniform size chunks
```
?

File: docs/CHUNKED_CHECKSUMS.md [hunk 2]
```diff
@@ -1,9 +1,4 @@
-# sha2-256-chunked
-
-## DRAFT multihash codec 0xb510
-
-> Hash of concatenated SHA2-256 digests of 8*2^n MiB source chunks
-> where n = ceil(log2(source_size / (10^4 * 8MiB))
+# CHUNKED CHECKSUMS
```
Comment (sir-sigurd) [inline docs/CHUNKED_CHECKSUMS.md:1]: not sure if using uppercase is justified

File: docs/FAQ.md [hunk 3]
```diff
@@ -65,6 +65,8 @@ a local machine or foreign region)—I/O is much faster.
 1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](api-reference/cli.md#quilt_transfer_max_concurrency)
 above its default to match your available vCPUs.
 
+1. If you are using Quilt Catalog 1.51 (released Feb 2024), you can enable CHUNKED_CHECKSUMS which will calculate the checksums in parallel, or reuse them if already existing in S3. Parallel checksums are also available by default in `quilt3` v6 or later (pre-released Feb 2024).
```
Comment (sir-sigurd) [inline docs/FAQ.md:68]:
```suggestion
1. If you are using Quilt Catalog 1.51 (released Feb 2024), you can enable CHUNKED_CHECKSUMS which will calculate the checksums in parallel, or reuse them if already existing in S3. Chunked checksums are also available by default in `quilt3` v6 or later (pre-released Feb 2024).
```

I'm not if you refer to the feature name or CloudFormation parameter name when you say `CHUNKED_CHECKSUMS`, but in either case spelling seems wrong

**Review by nl0 — APPROVED** [review]
seems ok in general, given Sergey's feedback is addressed

**Review by drernie — COMMENTED** [review]
Incorporating suggestions.

File: docs/FAQ.md (hunk 3 above)
Comment (sir-sigurd) [reply docs/FAQ.md:68]: how this was resolved?

**Comment by drernie** [comment]
I think that's all the changes.  @dimaryaz you can approve and merge if you need it for your next release...

File: docs/FAQ.md [hunk 4]
```diff
@@ -65,6 +65,8 @@ a local machine or foreign region)—I/O is much faster.
 1. Increase [`QUILT_TRANSFER_MAX_CONCURRENCY`](api-reference/cli.md#quilt_transfer_max_concurrency)
 above its default to match your available vCPUs.
 
+1. If you are using Quilt Catalog 1.51 (released Feb 2024), you can enable `ChunkedChecksums` which will calculate the checksums in parallel, or reuse them if already existing in S3. Parallel checksums are also available by default in `quilt3` v6 or later (pre-released Feb 2024).
```
Comment (sir-sigurd) [inline docs/FAQ.md:68]: I think it worth to say that `ChunkedChecksums` is CloudFormation parameter otherwise it's not clear how to enable it

**Review by sir-sigurd — DISMISSED** [review]

**Comment by drernie** [comment]
@sir-sigurd Sorry, I think you need to re-approve it after my last change.

**Review by sir-sigurd — APPROVED** [review]

**Pull request merged**